serverlessworkflow/modules/ROOT/pages/data-index/data-index-usecase-singleton.adoc

```diff
@@ -0,0 +1,183 @@
+= Deploying Data Index and {product_name} application on Minikube
+:compat-mode!:
+// Metadata:
+:description:  Deploying {product_name} application and Data Index on Minikube with operator
+:keywords: kogito, workflow, quarkus, serverless, kn, kubectl, minikube, operator, dataindex
+:table-caption: Data Set
+// envs for common content
+:kubectl_prereq: command-line tool is installed. Otherwise, Minikube handles it.
+//Common constants
+:data_index_ref: Data Index
+
+
+This document describes how to deploy a workflow application and the {data_index_ref} service using a local Kubernetes cluster, such as link:{minikube_url}[Minikube], using the link:{kogito_serverless_operator_url}[{operator_name}].
+
+For more information about Minikube and related system requirements, see link:{minikube_url}/docs/start/[Getting started with Minikube] documentation.
+
+This use case is intended to represent an installation with:
+
+* A singleton Data Index Service with PostgreSQL persistence
+* The `greeting` workflow (no persistence), that is configured to register events to the Data Index Service.
+
+// shared pre req
+include::common/_prerequisites.adoc[]
+
+You can check the Minikube installation by entering the following commands in a command terminal:
+
+.Verify Minikube version
+[source,shell]
+----
+minikube version
+----
+
+.Verify `kubectl` CLI version
+[source,shell]
+----
+kubectl version
+----
+
+[NOTE]
+====
+If `kubectl` is not installed, then Minikube handles it when you execute the following command:
+
+.`kubectl` is available using Minikube
+[source,shell]
+----
+alias kubectl="minikube kubectl --"
+----
+====
+
+.Procedure
+. Open a terminal and run the following commands
+. Create the namespace:
++
+--
+[source,shell]
+----
+kubectl create namespace usecase1
+----
+--
+
+. Deploy the {data_index_ref} Service and postgresql database:
++
+--
+include::common/_dataindex_deployment_operator.adoc[]
+
+Perform the deployments executing
+[source,shell]
+----
+kubectl kustomize infra/dataindex | kubectl apply -f - -n usecase1
+----
+
+----
+configmap/dataindex-properties-hg9ff8bff5 created
+secret/postgres-secrets-22tkgc2dt7 created
+service/data-index-service-postgresql created
+service/postgres created
+persistentvolumeclaim/postgres-pvc created
+deployment.apps/data-index-service-postgresql created
+deployment.apps/postgres created
+----
+
+Give some time for the data index to start, you can check that it's running by executing.
+
+[source,shell]
+----
+kubectl get pod -n usecase1
+----
+
+----
+NAME                                             READY   STATUS    RESTARTS       AGE
+data-index-service-postgresql-5d76dc4468-lb259   1/1     Running   0              2m11s
+postgres-7f78499688-lc8n6                        1/1     Running   0              2m11s
+----
+--
+. Deploy the workflow:
++
+--
+
+.Use case kustomization.yaml with resources that deploy the workflow inside usecases/usecase1 folder
```

Comment:
Same a previous guide, I do not see these created by user, let's tell him to create these files too.